### PR TITLE
Adding Version Number of taskcluster app to docs drawer.

### PR DESCRIPTION
**Pull request by iFlameing**
This is related to #453 in which we have to add the version number of taskcluster app to docs drawer.

### CODE REVIEW

File: ui/.neutrinorc.js [hunk 1]
```diff
@@ -153,6 +153,12 @@ module.exports = {
           .test(/\.all-contributorsrc$/)
           .use('json-loader')
             .loader('json-loader');
+
```
Comment (helfi92) [inline ui/.neutrinorc.js]: Remove empty spaces. Keep the newline however.

File: .git-version [hunk 2]
```diff
@@ -0,0 +1 @@
+8026691dab4045166c5a1798f3c7661ee622da4e
```
Comment (helfi92) [inline .git-version]: This file shouldn't be commited.

File: ui/src/components/Dashboard/index.jsx [hunk 3]
```diff
@@ -188,8 +197,26 @@ export default class Dashboard extends Component {
...
+    this.loadVersionNumber();
+  }
+
+  async loadVersionNumber() {
+    try {
+      const {
+        default: versionNumber,
+      } = await import(`../../../../.git-version`);
+
+      this.setState({ versionNumber });
```
Comment (helfi92) [inline ui/src/components/Dashboard/index.jsx]: Set the version in `componentDidMount`. `loadVersionNumber` should just return the value.

**Comment by iFlameing** [comment]
@helfi92 please! take a look, I made the changes as you suggested.

**Comment by iFlameing** [comment]
@helfi92 I made all the changes as you suggested above. But we are ignoring the error which will be thrown if we don't have .git-version file in root app directory  which I don't understand why? I also intended to add a class to nav-element but I was little bit afraid in changing code. thanks!

**Comment by helfi92** [comment]
>  But we are ignoring the error which will be thrown if we don't have .git-version file in root app directory which I don't understand why?

Good question. I added a comment to the issue to ask Dustin. Let's wait and see what he says.

**Comment by iFlameing** [comment]
@helfi92 rebased the pr :)

**Pull request merged**